spring-cloud-skipper-server-core/src/main/java/org/springframework/cloud/skipper/server/controller/ReleaseController.java

```diff
@@ -169,6 +170,18 @@ public ReleaseControllerLinksResource resourceLinks() {
 		return this.releaseResourceAssembler.toResource(release);
 	}
 
+	@RequestMapping(path = "/cancel/{name}", method = RequestMethod.POST)
+	@ResponseStatus(HttpStatus.OK)
+	public ResponseEntity<Void> cancel(@PathVariable("name") String releaseName) {
+		boolean accept = this.skipperStateMachineService.cancelRelease(releaseName);
+		if (accept) {
+			return new ResponseEntity<>(HttpStatus.ACCEPTED);
+		}
+		else {
+			return new ResponseEntity<>(HttpStatus.OK);
```

Comment:
I guess we could return `error` response instead of OK as the user request is not successfully completed here.

Reply:
Something I thought about but didn't do, What about if we'd return `CancelResponse` and then have fields in that pojo to indicate if event was sent to processing, and client would need to process that response? Then rest layer would work as is and if error is thrown in controller it has a real meaning for errors. Then whole request/response would be fully extendable.

Reply:
ok, I'll make these changes monday and then we can see how it looks like. This was a bit shitty PR but I was more focused on how it'd look from a shell perspective.